CUDA_UPGRADE_GUIDE.MD

```diff
@@ -36,40 +36,52 @@ https://developer.download.nvidia.com/compute/redist/cudnn/v8.3.2/local_installe
-    3. Create new repo to host manylinux-cuda images for example : https://hub.docker.com/r/pytorch/manylinux-cuda115 . And give bots write and read acceess to this repo. This step to be removed once following [issue](https://github.com/pytorch/builder/issues/901) is addressed
-    4. Push the images to Docker Hub. This step should be automated with the help with GitHub Actions in the `pytorch/builder` repo. Make sure to update the `cuda_version` to the version you're adding in respective YAMLs, such as `.github/workflows/build-manywheel-images.yml`, `.github/workflows/build-conda-images.yml`, `.github/workflows/build-libtorch-images.yml`.
-    5. Verify that each of the workflows that push the images succeed by selecting and verifying them in the [Actions page](https://github.com/pytorch/builder/actions/workflows/build-libtorch-images.yml) of pytorch/builder. Furthermore, check [https://hub.docker.com/r/pytorch/manylinux-builder/tags](https://hub.docker.com/r/pytorch/manylinux-builder/tags), [https://hub.docker.com/r/pytorch/libtorch-cxx11-builder/tags](https://hub.docker.com/r/pytorch/libtorch-cxx11-builder/tags), and [https://hub.docker.com/r/pytorch/conda-builder/tags](https://hub.docker.com/r/pytorch/conda-builder/tags) to verify that the right tags exist for all three types of images.
-
-## 3. Modify code to install the new CUDA for Windows
+## 2. Modify scripts to install the new CUDA for Conda Docker Linux containers.
+There are three types of Docker containers we maintain in order to build Linux binaries: `conda`, `libtorch`, and `manywheel`. They all require installing CUDA and then updating code references in respective build scripts/Dockerfiles.  This step is about conda
+
+1. Modifying [`install_cuda.sh`](common/install_cuda.sh) and build Conda Docker file:
+    1. Follow this [PR 992](https://github.com/pytorch/builder/pull/992) for all steps in this section
+    2. Find the CUDA install link [here](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&=Debian&target_version=10&target_type=runfile_local)
+    3. Get the cudnn link from NVIDIA on the PyTorch Slack
+    4. Run the `install_116` chunk of code on your devbox to make sure it works.
+    5. Check [this link](https://arnon.dk/matching-sm-architectures-arch-and-gencode-for-various-nvidia-cards/) to see if you need to add/remove any architectures to the nvprune list.
+    6. Go into your cuda-11.6 folder and make sure what you're pruning actually exists. Update versions as needed, especially the visual tools like `nsight-systems`.
+    7. Add setup for our Docker `conda` scripts/Dockerfiles
+    8. To test that your code works, from the root builder repo, run something similar to `export CUDA_VERSION=11.3 && ./conda/build_docker.sh` for the `conda` images. 
```

Comment:
```suggestion
    8. To test that your code works, from the root builder repo, run something similar to `export CUDA_VERSION=11.6 && ./conda/build_docker.sh` for the `conda` images. 
```
Maybe switch out all the 11.blah to 11.x for better upkeep though CUDA 12 might be happening soon so maybe not

Reply:
lets keep 11.6 references for now. I want to update the readme to all latest PRs